src/include/homestore/index_service.hpp

```diff
@@ -32,7 +32,10 @@ class VirtualDev;
 class IndexServiceCallbacks {
 public:
     virtual ~IndexServiceCallbacks() = default;
-    virtual std::shared_ptr< IndexTableBase > on_index_table_found(const superblk< index_table_sb >& cb) = 0;
+    virtual std::shared_ptr< IndexTableBase > on_index_table_found(superblk< index_table_sb > const&) {
+        assert(0);
```

Comment:
What is the reason to provide a default that abends?

Reply:
I did this earlier to avoid caller defining the IndexServiceCallbacks even when they are not using Index service, but I don't think this is needed, will revert back.